packages/firebase_data_connect/firebase_data_connect/protos/google/api/README.md

```diff
@@ -0,0 +1,36 @@
+## API Protos
+
+This folder contains the schema of the configuration model for Google's
+internal API serving platform, which handles routing, quotas, monitoring,
+logging, and the like.
+
+Google refers to this configuration colloquially as the "service config",
+and the `service.proto` file in this directory is the entry point for
+understanding these.
+
+## Using these protos
+
+To be honest, we probably open sourced way too much of this (basically by
```

Comment:
I'd like to understand why we have to include this protos/google/api folder in our sdk project. Is there any other way to reference this? If we have to include this in, can we just take in the necessary subset?

Reply:
We need the protos to make sure that we can talk to the gRPC endpoints. I ended up removing the unnecessary protos